setup.py

```diff
@@ -81,5 +81,9 @@ def get_version():
         classifiers=classifiers,
         python_requires=">=3.8",
         install_requires=install_reqs,
-        extras_requires=dict(dask=["dask[all]"]),
+        extras_requires=dict(
+            dask=["dask[all]"], 
+            histogram=["xhistogram"], 
+            complete=["dask[all]", "xhistogram"]
```

Comment:
Should it be? Or does `complete` have a special meaning

```suggestion
            all=["dask[all]", "xhistogram"]
```